client/actions/planning.js

```diff
@@ -221,15 +230,20 @@ const openPlanningEditor = (planning) => ({
  * @return arrow function
  */
 const openPlanningEditorAndAgenda = (planning) => (
-    (dispatch, getState) => {
-        const agenda = selectors.getAgendas(getState()).find(
-            (a) => (a.planning_items || []).indexOf(planning) > -1
-        )
-        if (agenda && agenda._id !== selectors.getCurrentAgendaId(getState())) {
-            dispatch(selectAgenda(agenda._id))
+    (dispatch, getState, { notify, $timeout }) => {
+        if (selectors.getPrivileges(getState()).planning_planning_management === 1) {
```

Comment:
Since we are going to check privileges in many actions, maybe we could create a function that check the permission which we would use to wrap the actual actions when we export them ?
```js
export const openPlanningEditorAndAgenda = checkPermission('planning_planning_management', _openPlanningEditorAndAgenda)
```
This is just a suggestion but in this way we would avoid this extra condition and logic in many actions. Also we would be able to see the relationship between actions and permissions in a more clear way, all in the same part of code.
what do you think ?